watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab.py

```diff
@@ -0,0 +1,622 @@
+from idaes.generic_models.unit_models import Product
+import idaes.core.util.scaling as iscale
+from idaes.generic_models.costing import UnitModelCostingBlock
+
+from watertap.core.util.initialization import assert_degrees_of_freedom
+
+from watertap.core.wt_database import Database
+import watertap.core.zero_order_properties as prop_ZO
+from watertap.unit_models.zero_order import (
+    FeedZO,
+    MetabZO,
+)
+from watertap.core.zero_order_costing import ZeroOrderCosting
+
+
+def main():
+    m = build()
+
+    set_operating_conditions(m)
+    assert_degrees_of_freedom(m, 0)
+    assert_units_consistent(m)
+
+    initialize_system(m)
+
+    results = solve(m)
+    display_results(m)
+
+    add_costing(m)
+    initialize_costing(m)
+    assert_degrees_of_freedom(m, 0)
+
+    m.fs.metab_hydrogen.hydraulic_retention_time.fix(6)  # default - 12 hours, 0.5x
+    m.fs.metab_hydrogen.generation_ratio["cod_to_hydrogen", "hydrogen"].set_value(
+        0.05
+    )  # default - 0.005, 10x
+    m.fs.costing.metab.bead_bulk_density["hydrogen"].fix(7.17)  # default 23.9, 0.3x
+    m.fs.costing.metab.bead_replacement_factor["hydrogen"].fix(1)  # default 3.376, 0.3x
+    m.fs.metab_hydrogen.energy_electric_mixer_vol.fix(0.049875)  # default 0.049875
+    m.fs.metab_hydrogen.energy_electric_vacuum_flow_vol_byproduct.fix(
+        9.190
+    )  # default 9190, 0.001x
+    m.fs.metab_hydrogen.energy_thermal_flow_vol_inlet.fix(7875)  # default 78750, 0.1x
+    m.fs.costing.metab.bead_cost["hydrogen"].fix(14.40)  # default 1440, 0.01x
+    m.fs.costing.metab.reactor_cost["hydrogen"].fix(78.9)  # default 789, 0.1x
+    m.fs.costing.metab.vacuum_cost["hydrogen"].fix(5930)  # default 59300, 0.1x
+    m.fs.costing.metab.mixer_cost["hydrogen"].fix(27.40)  # default 2740, 0.1x
+    m.fs.costing.metab.membrane_cost["hydrogen"].fix(498)  # default 498
+
+    m.fs.metab_methane.hydraulic_retention_time.fix(15)  # default 150, 0.1x
+    m.fs.metab_methane.generation_ratio["cod_to_methane", "methane"].set_value(
+        0.101
+    )  # default 0.101, no change
+    m.fs.costing.metab.bead_bulk_density["methane"].fix(7.17)  # default 23.9, 0.3x
+    m.fs.costing.metab.bead_replacement_factor["methane"].fix(1)  # default 3.376, 0.3x
+    m.fs.metab_methane.energy_electric_mixer_vol.fix(0.049875)  # default 0.049875
+    m.fs.metab_methane.energy_electric_vacuum_flow_vol_byproduct.fix(
+        1.53
+    )  # default 15.3, 0.1x
+    m.fs.metab_methane.energy_thermal_flow_vol_inlet.fix(0)  # default 0
+    m.fs.costing.metab.bead_cost["methane"].fix(14.40)  # default 1440, 0.01x
+    m.fs.costing.metab.reactor_cost["methane"].fix(78.9)  # default 789, 0.1x
+    m.fs.costing.metab.vacuum_cost["methane"].fix(136.0)  # default 1360, 0.1x
+    m.fs.costing.metab.mixer_cost["methane"].fix(27.40)  # default 2740, 0.1x
+    m.fs.costing.metab.membrane_cost["methane"].fix(498)  # default 498
+
+    results = solve(m)
+    display_costing(m)
+
+    print(
```

Comment:
Similarly, why is this not part of the `display_results` method (or a similar sub-method)?